vizro-core/hatch.toml

```diff
@@ -65,6 +65,7 @@ test-integration = [
   "pytest tests/integration --headless -k yaml {args}"
 ]
 test-unit = "pytest tests/unit {args}"
+test-js = "echo '' && echo 'To run tests for the javascript files you need to:' && echo '1. Install Node.js by downloading the LTS version at https://nodejs.org/en/download' && echo '2. Run command: \"npm install\" (to install npm dependencies)' && echo '3. Run command: \"npx jest\" (to run javascript tests)' && echo ''"
 update-snyk-requirements = "python ../tools/generate_snyk_requirements.py {args}"
```

Comment:
As discussed, this should cater for both cases, if setup is complete, then run tests, if not, show these instructions